android/pytorch_android/CMakeLists.txt

```diff
@@ -134,6 +136,8 @@ if(ANDROID_ABI)
       libeigen_blas
       libcpuinfo
       libclog
+      libfmt
```

Comment:
Same as above, fmt is header only lib, isn't it?

Reply:
updated